### PR TITLE
fix: add docker logs

### PR DESCRIPTION
**Pull request by ajasnosz**
# Description

Adds script to set up logging to Splunk from docker compose deployment.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix
- [ ] New feature
- [x] Refactor/improvement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

manual test

## Checklist

- [x] My commit message is [conventional](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] I have run pre-commit on all files before creating the PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: docs/dockercompose/9-splunk-logging.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Logging
+
+The default configuration of docker compose is not sending the logs to Splunk. Container logs can be accessed with command:
+```
+docker logs <container_name/id>
+```
+
+Creating a logs requires updating configuration of several docker compose files. To simplify this process, inside the
```
Comment (wojtekzyla) [inline docs/dockercompose/9-splunk-logging.md:8]: creating logs

File: docs/dockercompose/9-splunk-logging.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Running script requires installation of `ruamel.yaml` package for python. It can be done with command:
+```
+pip3 install ruamel.yaml
+```
+
+The following parameters have to be configured in `.env` file:
+`SPLUNK_HEC_TOKEN`, `SPLUNK_HEC_PROTOCOL`, `SPLUNK_HEC_HOST`, `SPLUNK_HEC_PORT`, `SPLUNK_LOG_INDEX`, `SPLUNK_HEC_INSECURESSL`.
+
+More about `.env` configuration can be found in [.env file configuration](./6-env-file-configuration.md).
```
Comment (wojtekzyla) [inline docs/dockercompose/9-splunk-logging.md:21]: maybe we could add description of these variables in `/6-env-file-configuration.md` as well?

**Review by wojtekzyla — CHANGES_REQUESTED** [review]

File: docs/dockercompose/9-splunk-logging.md (hunk 2 above)
Comment (ajasnosz) [reply docs/dockercompose/9-splunk-logging.md:21]: good point

**Review by wojtekzyla — APPROVED** [review]

**Pull request merged**